### ISSUE TITLE
Rework Attestation section

### ISSUE THREAD

**Issue by fitzthum** [confidential-containers/confidentialcontainers.org#54]
We have a section about attestation on the website. Currently it just has an old clone of the docs from the Trustee repo. This isn't very use friendly. We should update it with a foolproof guide to using Trustee with CoCo.

We should probably pick just one installation method to feature here. I am thinking that we might want to show how to use the Trustee operator, but I'm not sure. Any ideas? @bpradipt @Xynnn007 @lmilleri

**Comment by Xynnn007** [comment]
Yes. [The page](https://confidentialcontainers.org/docs/attestation/attestation-service/) looks still abstract. Besides the operator things, It would make sense to write some quick starts for AS & KBS deployment and usage, probably after the EAR PR as it would somehow change the behavior of AS. Certainly I can help with this.

**Comment by fitzthum** [comment]
One note is that we currently have [this page](https://confidentialcontainers.org/docs/attestation/key-broker-service/kbs-backed-by-akv/). We should keep this functionality available but I'm not sure how it will work with the operator.
